### ISSUE TITLE
md_5.bungee.protocol.EntityEffect 79

### ISSUE THREAD

**Issue by Keaton1188** [PaperMC/Travertine#102]
Hi there, having this strange issue on our new FTB Builders Paradise Server, it is kicking me with a md_5 protocol Entity Effect error and doesn't let me back in. Not sure if this is pack related to Travertine or the Mod pack.

This is the error from Travertine Console. 

![Untitled](https://user-images.githubusercontent.com/24898745/57000296-d7c2f000-6baa-11e9-8bc2-02ded03c39d3.png)

**Comment by 123Kappa** [comment]
Did u find the cause of this?

**Comment by y6x2digc** [comment]
I can confirm this

**Comment by Irgendwer01** [comment]
@electronicboy is there any Solution for this issue?

**Comment by electronicboy** [comment]
No, because I've not had the investment to rip apart 100 mod packs to work out what some mod is doing to the packet, or if the behavior causing this is still applicable to "modern" versions of forge or even at all, https://github.com/PaperMC/Waterfall/issues/418

**Comment by joeydoesthings** [comment]
I have the same issue. Right after I died in game, it wouldn't let me reconnect. it says the server is down, but it clearly isn't.

**Comment by Irgendwer01** [comment]
@electronicboy after some research and testing i found that the mod Just Enough IDs is sending the packet,the mod expands the max IDs can be used

**Comment by astei** [comment]
Judging from JEID's resource code, it extends [the potion effect packets in a non-standard way](https://github.com/DimensionalDevelopment/JustEnoughIDs/blob/d095e791ae474e813d69bfcfaa6d1861f1ac178f/src/main/java/org/dimdev/jeid/JEIDTransformer.java#L123) which breaks BungeeCord and Waterfall because of a fix to a [Forge 1.8 issue](https://github.com/PaperMC/Waterfall/blob/master/BungeeCord-Patches/0032-Fix-potion-race-condition-on-Forge-1.8.9.patch).

I see three options: complain that JEID is non-standard (not really an option), drop the 1.8.9 bugfix (since Minecraft 1.8 is over 6 years old and needs to go away) or flat-out say that JEID is not supported on Waterfall/Travertine and to use Velocity instead (which does not intercept the packet and doesn't have the issue).

**Comment by Irgendwer01** [comment]
@astei it happens when dying too so i don't think it's directly potion related

**Comment by electronicboy** [comment]
The packet is the one used for potion effects

**Comment by Irgendwer01** [comment]
isn't there any way to let Travertine ignore this packet?

**Comment by Xernium** [comment]
Well yes, disable entity meta rewriting in the config. If you still have issues then I’d suggest you try using Velocity instead of Travertine. I’m sure that if the config option doesn’t fix it, Velocity won’t have the issue at all
